[BUG] Unexpected error when failed to download the dataset

**Describe the bug**

Looks like a bug: https://github.com/alibaba/GraphScope/actions/runs/3140643667/jobs/5102563433

```python
            max_retries = gs_config.dataset_download_retries
            error_msg_tpl = "URL fetch failure on {}:{} -- {}"
            try:
                for retry in range(max_retries):
                    backoff = max(2**retry, 1.0)
                    try:
                        urlretrieve(origin, fpath, show_progress)
                    except urllib.error.HTTPError as e:
                        error_msg = error_msg_tpl.format(origin, e.code, e.msg)
                        logger.warning("{0}, retry {1} times...".format(error_msg, retry))
                        if retry >= max_retries - 1:
                            raise Exception(error_msg)
                        time.sleep(backoff)
                    except urllib.error.URLError as e:
                        # `URLError` has been made a subclass of OSError since version 3.3
                        # https://docs.python.org/3/library/urllib.error.html
>                       error_msg = error_msg.format(origin, e.errno, e.reason)
E                       UnboundLocalError: local variable 'error_msg' referenced before assignment
```

**Additional context**
Add any other context about the problem here.

--- a/python/graphscope/dataset/io_utils.py
+++ b/python/graphscope/dataset/io_utils.py
@@ -266,7 +266,7 @@ def download_file(  # noqa: C901
                 except urllib.error.URLError as e:
                     # `URLError` has been made a subclass of OSError since version 3.3
                     # https://docs.python.org/3/library/urllib.error.html
-                    error_msg = error_msg.format(origin, e.errno, e.reason)
+                    error_msg = error_msg_tpl.format(origin, e.errno, e.reason)
                     logger.warning("{0}, retry {1} times...".format(error_msg, retry))
                     if retry >= max_retries - 1:
                         raise Exception(error_msg)

